x/auth/vesting/cmd/vestcalc/README.md

```diff
@@ -32,20 +32,36 @@ tool for other options.
 
 ## Writing a schedule
 
-When the `--write` flag is set, the tool will write a schedule in JSON to
-stdout. The following flags control the output:
+When the `--write` flag is set, the tool will write a schedule to stdout.
+The format is the the format expected by the `create-periodic-vesting-account`
+or `create-clawback-vesting-account` cli commands, namely a JSON
+object with the members:
 
-- `--coins:` The coins to vest, e.g. `100ubld,50urun`.
-- `--months`: The number of months to vest over.
-- `--time`: The time of day of the vesting event, in 24-hour HH:MM format.
-  Defaults to midnight.
+- `"start_time"`: integer UNIX time;
```

Comment:
I'll stick with "UNIX time" for the concept, since a) it matches the the Go `time` package API (which doesn't mention an epoch, only "UNIX time"), and b) "UNX time" has its own wikipedia page - "POSIX time" redirects there.